saleor/payment/gateways/braintree/plugin.py

```diff
@@ -84,8 +84,9 @@ class BraintreeGatewayPlugin(BasePlugin):
             "label": "Require 3D secure",
         },
         "Supported currencies": {
-            "type": ConfigurationTypeField.LIST,
-            "help_text": "Determines currencies supported by gateway.",
+            "type": ConfigurationTypeField.STRING,
+            "help_text": "Determines currencies supported by gateway."
+            " Please enter currencies separated by a comma.",
```

Comment:
Maybe use "currency codes" to make it more clear:

```suggestion
            " Please enter currency codes separated by a comma.",
```